### ISSUE TITLE
Import statement is not valid (.gql modules)

### ISSUE THREAD

**Issue by Crainov** [dotansimha/graphql-code-generator#3219]
**Describe the bug**
 
After update to ver 1.10.0 stop working import in gql files, I'm use graphql-tag loader for importing. Maybe this [commit](https://github.com/ardatan/graphql-toolkit/commit/a18a44218e2ba95795595130d3ea2efa44cba3aa#diff-d9cf34190de684ddaa13adf9181f31ab) is the source of issues 

**To Reproduce**
`graphql-codegen`

**Result**
```
Error: 
          Import statement is not valid: import '@shared/components/UpperSection/upperSection.gql'
          If you want to have comments starting with '# import', please use ''' instead!
          You can only have 'import' statements in the following pattern;
          # import [Type].[Field] from [File]
    
        at parseImportLine (/Users/max/Documents-local/InterfirstWebsite/node_modules/@graphql-toolkit/core/index.cjs.js:213:15)
```

1. My GraphQL operations:

```
#import '@shared/components/UpperSection/upperSection.gql'
#import '@shared/components/CustomerStories/customerStoriesSection.gql'
#import '@components/Landing/WhoWeAre/whoWeAreSection.gql'
#import '@components/Landing/AboutUsSection/aboutUsSection.gql'
#import '@components/Landing/BuyArea/buyHomeSection.gql'

query MainPage ($pathname: String) {
  pageEntity(where: { pathname: $pathname }) {
    ...UpperSection
    ...CustomerStoriesSection
    ...WhoWeAreSection
    ...AboutUsSection
    ...BuyHomeSection
  }
}
```

2. My `codegen.yml` config file:

```yml
schema: https://api-useast.graphcms.com/v1/
documents: /*/**.gql
generates:
  ./@generatedTypes/index.ts:
    plugins:
      - 'typescript'
      - 'typescript-operations'

```

**Environment:**

- OS: Mac OSX
- `"@graphql-codegen/cli": "1.10.0"`: 
- NodeJS: v10.4.0

**Additional context**

**Comment by ardatan** [comment]
Could you try with this canary version? `1.10.1-alpha-7435fca2.6`

**Comment by Crainov** [comment]
@ardatan yes import statement bug fixed, but I faced with another issue connected with webpack aliases, they stopped working.
`Load GraphQL schemas
      ✖ Load GraphQL documents
        → Cannot find module '@shared/components/UpperSection/upperSection.gql'
        Generate`

**Comment by ardatan** [comment]
@Crainov graphql-code-generator supports `graphql-import` but it doesn't know your custom configuration obviously. So you can disable `graphql-import` parsing with `skipGraphQLImport` option.
```yml
documents:
  - './src/*.gql':
    skipGraphQLImport: true
```

**Comment by Crainov** [comment]
@ardatan ah, ok! thank you

**Comment by dotansimha** [comment]
Fixed in 1.11.0
